assets/scripts/aircraft.js

```diff
@@ -996,7 +1003,12 @@ var Aircraft=Fiber.extend(function() {
 
       if(response.length >= 1) {
         if(response_end) response_end = ", " + response_end;
-        ui_log(this.getRadioCallsign() + ", " + response.join(", ") + response_end);
+        
+        var r_log = function f(){var x = [];for(var i=0;i<response.length;i++){x.push(response[i].log)}return x;};
+        var r_say = function f(){var x = [];for(var i=0;i<response.length;i++){x.push(response[i].say)}return x;};
```

Comment:
This should be writeable as:

    var r_log = response.map(function (e) {return e.log}).join(", ");
    var r_say = response.map(function (e) {return e.say}).join(", ");